storefront-reference-architecture-master/cartridges/app_storefront_base/cartridge/templates/default/product/productDetails.isml

```diff
@@ -121,6 +122,9 @@
                             </div>
 
                             <!-- Cart and [Optionally] Apple Pay -->
+                            <isif condition="${product.showPromo}">
```

Comment:
What about the other case?

Reply:
Do we need to add it because it says to add text when is true.